client/components/domains/use-my-domain/utilities/get-option-info.js

```diff
@@ -137,12 +132,13 @@ export function getOptionInfo( {
 				),
 			};
 			break;
-		default:
-			transferContent = optionInfo.transferNotSupported;
-	}
-
-	if ( ! isDomainTransferrable ) {
-		transferContent = domainTransferContent;
+		default: {
+			const availabilityNotice = getAvailabilityNotice( domain, availability );
```

Comment:
This should be just the `status` field of the availability object.
```suggestion
			const availabilityNotice = getAvailabilityNotice( domain, availability.status );
```